doc/xbgpu.design.b.rst

```diff
@@ -0,0 +1,57 @@
+Beamforming
+===========
+
+GPU kernel
+----------
+For the low number of beams required for MeerKAT (8 single-pol beams), the
+beamforming operation is entirely bandwidth-limited. There is thus no benefit
```

Comment:
Minor gripe/request to be more explicit about what bandwidth we/you're referring to here. 
* I know context etc should explain it, but 
* I did Ctrl+F through the docs for *bandwidth* and 
* almost all usages have some explicit context mentioned near/next to the word usage (e.g. 'DRAM bandwidth', or 'bandwidth of the network interface').